### DISCUSSION TITLE
support for bun, Deno?

### DISCUSSION THREAD

**Discussion by Habi-Thapa** [moleculerjs/moleculer#1312] [General]
hi, is moleculer nicely integrated with bun? We have the code from 3-4 years back running on nodejs. need to optimize the performance thinking of switching to bun for performance? should we switch to deno or bun or not switch at all? Can some old library cause conflict if we change? if for old code it's not good idea what about new projects?

**Comment by icebob** [comment]
There is no official Deno or Bun version of Moleculer but it doesn't mean it's not working. You should try and test it.
